ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+Also, it is important to acknowledge that there are developers who are also functioning as SREs, and in fact this is definitely in the spirit of Site Reliability Engineering in general. An example may be a smaller Engineering team who fill many roles, or maybe a newer service that is early in its stages of maturity. However, for the purpose of these personas, consider the person acting in the role of a "Developer" when performing that function, and similarly acting in the role of an SRE when performing that function.
+
+**Engineering/Architecture Lead**
+
+The Developers and SREs do not work in isolation. To obtain the economies of scale that are necessary, and to make sure that work is not duplicated, they need to conform to the 'systems (including processes)' and 'architecture' that best suits the business goal. This is the primary role of this persona. They provide the "guard-rails" for the developers, and help keep the services aligned with accepted strategic technical direction. They also have a key role in developing best practices, and coaching teams to avoid anti-patterns.
+
+**Quality Engineer**
+
+Service development has a life-cycle that ultimately results in a service which is deployed and used by customers. The quality phase of that life-cycle is absolutely critical in the reliability of that service, as SREs need to be able to trust what is being deployed. That is the role for this persona. It is important to note, that this is not the role of the SRE, especially testing in Production environments.
+
+**Site Reliability Engineer (SRE)**
+
+The Site or Service Reliability Engineer is an essential part of the service. Without the SRE, the service is incomplete. Their primary focus is ensuring that the services are "running". When there is a problem, it is up to them to resolve the issue as quickly as possible. However, this is only the end goal. The role of an SRE is to contribute to the reliability of the service so that the need to fix something is the exception rather than the rule. This is best epitomized in this quote by one of Red Hat's SRE Managers: "Do not accept repeated failure". That is, if something fails (e.g. SRE is alerted), then follow through with all "actions" so that the problem does not repeat. These actions can be things like: from identifying the issue with the responsible engineering team through to actually fixing the problem in code, personally. 
+
+In many organizations, SRE functions support more than one service. Many projects undertaken by SRE teams provide the tooling, workflows and platforms needed to operate at scale. A key metric is for the SRE team to grow sub-linearly with "hockey stick" customer growth while minimizing toil and technical debt.
```

Comment:
Every role has these in common as well.  It not exclusive to SRE to support more than one "service", provide e2e operational contributions, nor attempt to minimize toil or tech debt.

Reply:
That is very true @candita - considering that this is a common "theme" which influences every persona, maybe we need to start with an introductory paragraph which expresses this thought (and also ties into other "Operate First" initiatives such as "Everyone-owns-the-SLO" :) ). WDYT?

Reply:
Thanks @candita . I take your point, and agree that we can focus on the differences. Thus I have done a re-wording of this section to reflect this:

```
The Site or Service Reliability Engineer is an essential part of the service. Of course, a primary focus is to ensure that the services are "running" (i.e. operating as intended). Mostly, that is through attention to, and action of, alerts when they are firing and making sure everything (being monitored) is "green". However, this is only one aspect of the value that SREs bring. The key word in the title of this persona, is "reliability". SREs are also valuable consultants, particularly in these key areas: observability; automation; resilience; toil prevention; capacity planning; and, operability. They are most familiar with the operational boundaries of the services they support, and they own the core of the incident and problem management processes and are accountable for it and their continuous improvement. They can best articulate the problems being encountered since they are hands-on.

In many organizations, SRE functions support more than one service. Many projects undertaken by SRE teams provide the tooling, workflows and platforms needed to operate at scale. A key metric is for the SRE team to grow sub-linearly with "hockey stick" customer growth while minimizing toil and technical debt.
```

WDYT?